README.md

```diff
@@ -12,270 +12,298 @@ Web5 consists of the following components:
 
 The SDK sets out to gather the most oft used functionality from all three of these 
 pillar technologies to provide a simple library that is as close to effortless as 
-possible. 
+possible.
 
 ## Docs
 
-[API Reference](https://tbd54566975.github.io/web5-js/)
+### Installation
 
-### **`new Web5([options])`**
-
-Creates an isolated API object for doing Web5 things, split into three main top-level objects: `did`, `dwn`, and `vc`.
+```bash
+npm install @tbd54566975/web5
+```
```

Comment:
might be worth including that it's also available via the cdn link for people who don't want to use bundlers